This fails like

======================================================================
ERROR: test_gbuildtojson (gbuildtojson.CheckGbuildToJsonModules)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/data/rene/git/LibreOffice/libreoffice-6-0/solenv/qa/python/gbuildtojson.py", line 135, in test_gbuildtojson
    shutil.copytree(os.path.join(os.environ['SRCDIR'], module), os.path.join(self.tempsrc, module))
  File "/usr/lib/python3.6/shutil.py", line 359, in copytree
    raise Error(errors)
shutil.Error: [('/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent.browser', '/tmp/gbuildm7c1xu2_/xmlsecurity/qa/unit/signing/data/S.gpg-agent.browser', "[Errno 6] No such device or address: '/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent.browser'"), ('/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent.extra', '/tmp/gbuildm7c1xu2_/xmlsecurity/qa/unit/signing/data/S.gpg-agent.extra', "[Errno 6] No such device or address: '/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent.extra'"), ('/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent', '/tmp/gbuildm7c1xu2_/xmlsecurity/qa/unit/signing/data/S.gpg-agent', "[Errno 6] No such device or address: '/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent'"), ('/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent.ssh', '/tmp/gbuildm7c1xu2_/xmlsecurity/qa/unit/signing/data/S.gpg-agent.ssh', "[Errno 6] No such device or address: '/data/rene/git/LibreOffice/libreoffice-6-0/xmlsecurity/qa/unit/signing/data/S.gpg-agent.ssh'")]

----------------------------------------------------------------------
Ran 2 tests in 120.312s

FAILED (errors=1)

Error: a unit test failed, please do one of:

make PythonTest_solenv_python CPPUNITTRACE="gdb --args"
    # for interactive debugging on Linux
make PythonTest_solenv_python VALGRIND=memcheck
    # for memory checking
make PythonTest_solenv_python DEBUGCPPUNIT=TRUE
    # for exception catching

You can limit the execution to just one particular test by:

make CPPUNIT_TEST_NAME="testXYZ" ...above mentioned params...

/data/rene/git/LibreOffice/libreoffice-6-0/solenv/gbuild/PythonTest.mk:38: recipe for target '/data/rene/git/LibreOffice/libreoffice-6-0/workdir/PythonTest/solenv_python/done' failed

We don't really need to test gbuild here, so just disable it..

diff --git a/solenv/Module_solenv.mk b/solenv/Module_solenv.mk
index 70eb5f9fb7c3..435f5c1d2549 100644
--- a/solenv/Module_solenv.mk
+++ b/solenv/Module_solenv.mk
@@ -11,7 +11,6 @@ $(eval $(call gb_Module_Module,solenv))
 
 $(eval $(call gb_Module_add_targets_for_build,solenv,\
 	Executable_concat-deps \
-	Executable_gbuildtojson \
 ))
 
 ifeq ($(COM),MSC)
@@ -22,19 +21,6 @@ $(eval $(call gb_Module_add_targets,solenv,\
 ))
 endif
 
-ifneq ($(DISABLE_PYTHON),TRUE)
-ifneq ($(MAKE_VERSION),3.81) # gbuildtojson requires 3.82+
-ifneq ($(OS),WNT) # disable on Windows for now, causes gerrit/jenkins failures
-ifneq ($(OS),MACOSX) # disable on macOS too, fails at least for me and would be pointless anyway surely
-$(eval $(call gb_Module_add_subsequentcheck_targets,solenv,\
-	CustomTarget_gbuildtesttools \
-	PythonTest_solenv_python \
-))
-endif
-endif
-endif
-endif
-
 ifneq ($(COMPILER_PLUGINS),)
 ifeq ($(COMPILER_EXTERNAL_TOOL)$(COMPILER_PLUGIN_TOOL),)
 $(eval $(call gb_Module_add_check_targets,solenv, \
